connectivity/lorawan/tests/UNITTESTS/doubles/CMakeLists.txt

```diff
@@ -34,8 +34,12 @@ target_sources(mbed-stubs-lorawan
 
 target_link_libraries(mbed-stubs-lorawan
     PRIVATE
-        mbed-headers
-        mbed-headers-lorawan
+        mbed-headers-base
+        mbed-headers-events
+        mbed-headers-hal
         mbed-stubs-headers
```

Comment:
This is for headers in UNITTESTS/stub/, mainly the scattered .h files that are not grouped into any subdirectories. Is this still needed here?

Reply:
Yes, All these headers libraries are required for lorawan stubs to build

Reply:
Cool, guess we can only remove it when when all components have been refactored